Makefile

```diff
@@ -1,24 +1,35 @@
+_TEST_PROJECT = myproject
+
 start-openshift-with-catalog:
+	@echo "---- Starting OpenShift ----"
 	oc cluster up --service-catalog
-	oc login -u system:admin
-	oc adm policy add-cluster-role-to-user cluster-admin developer
-	oc login -u developer -p developer
-	oc project openshift
-	oc adm policy add-cluster-role-to-group system:openshift:templateservicebroker-client system:unauthenticated system:authenticated
-.PHONY: start-openshift-with-catalog
 
-start-openshift:
-	oc cluster up
+	@echo "---- Granting admin rights to Developer ----"
 	oc login -u system:admin
 	oc adm policy add-cluster-role-to-user cluster-admin developer
```

Comment:
Why do you need to do this still? I did some testing and I was able to run clusters, store stuff...etc without the need for this.

Reply:
This makes it super easy to test different Service Brokers (most of them require to put stuff in `openshift` namespace). Additionally it makes debugging much easier, you can run `oc get pods --all-namespaces` for example.

BTW, why can't you use `oc cluster up && make install-templates` combo?

Reply:
Sure, I can use these instructions locally, but I couldn't get this set up to work with OpenShifter for example, where OpenShift is running on top of say Google Cloud rather than locally. `system:admin` is not so easily accessible when running remotely.

You can still however add the templates to developer's `My Project`, which IMO is just fine from the perspective of the templates here.

Testing different Service Brokers is something you might do yourself but I don't yet have any interest on that, and I doubt others that want to demonstrate Infinispan on top of OpenShift want right now.

Reply:
The only thing that I could be interested here would be maybe having the Infinispan templates appear in the home page. Do you need admin rights to do that?

Reply:
Ok.. I'll regroup makefile targets so that `install-templates-in-openshift-namespace` tries to obtain cluster admin rights (which BTW are necessary to do this).